main_test.go

```diff
@@ -0,0 +1,126 @@
+// Copyright 2018-2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//	http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"io/ioutil"
+	"net"
+	"net/http"
+	"os"
+	"os/exec"
+	"syscall"
+	"testing"
+	"time"
+
+	models "github.com/firecracker-microvm/firecracker-go-sdk/client/models"
+)
+
+var kernelImagePath = "./vmlinux"
+var rootDrivePath = "/dev/null"
+
+func init() {
+	if v := os.Getenv("KERNELIMAGE"); len(v) != 0 {
+		kernelImagePath = v
+	}
+
+	if v := os.Getenv("ROOTDRIVE"); len(v) != 0 {
```

Comment:
Do you really need the root drive to be overridden if all the tests are fine just using `/dev/null`?